exercises/saddle-points/tests/saddle-points.rs

```diff
@@ -0,0 +1,68 @@
+extern crate saddle_points;
+
+use saddle_points::*;
+
+#[test]
+fn identify_single_saddle_point() {
+    let input: Vec<Vec<u64>> = vec![vec![9, 8, 7],
+                                    vec![5, 3, 2],
+                                    vec![6, 6, 7]];
+    assert_eq!(vec![(1,0)], find_saddle_points(input));
+}
+
+
+#[test]
+#[ignore]
+fn identify_empty_matrix() {
+    let input: Vec<Vec<u64>> = vec![vec![]];
+    let expected: Vec<(u64, u64)> = Vec::new();
+
+    assert_eq!(expected, find_saddle_points(input));
+}
+
+#[test]
+#[ignore]
+fn identify_missing_saddle_point() {
+    let input: Vec<Vec<u64>> = vec![vec![1, 2, 3],
+                                    vec![3, 2, 1],
```

Comment:
If this test is based on the canonical test "Can identify lack of saddle points when there are none", this row of input data diverges from the [equivalent row in the canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/saddle-points/canonical-data.json#L39). Is there a reason for this?

Reply:
No, it was a typo. :-)